### ISSUE TITLE
Is there an equivalent function to VSCode's navigateToLastEditLocation?

### ISSUE THREAD

**Issue by domi91c** [neoclide/coc.nvim#3650]
**Is your feature request related to a problem? Please describe.**
I'd like to be able to jump to recent edits in my project, like you can do in VSCode with ⌘K ⌘Q.

**Describe the solution you'd like**
Functions to jump between recent edits in chronological order, like NextEditLocation, PreviousEditLocation. I don't know how VSCode decides how granular an "edit" is though, or how CoC should do that.

**Describe alternatives you've considered**
Maybe a plugin outside of CoC would work, but I feel like this makes sense to be included with CoC.

**Additional context**
[VSCode docs for navigateToLastEditLocation](https://code.visualstudio.com/updates/v1_28#_navigate-to-last-edit-location)

**Comment by chemzqm** [comment]
No, we don't track every edit but vim have change list `:h changelist`.
My suggestion is request vim or neovim for change list of all buffers.
